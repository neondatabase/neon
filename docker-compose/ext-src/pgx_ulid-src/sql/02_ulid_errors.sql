-- Test ULID error handling

-- Test invalid ULID string (too short)
SELECT '01GV5PA9EQG7D82Q3Y4PKBZSY'::ulid;

-- Test invalid ULID string (invalid character)
SELECT '01GV5PA9EQG7D82Q3Y4PKBZSYU'::ulid;

-- Test NULL handling
SELECT 'NULL to ulid conversion returns NULL' as test_name,
       NULL::ulid IS NULL as result;

