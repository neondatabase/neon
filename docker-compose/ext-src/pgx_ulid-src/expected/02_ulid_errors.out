-- Test ULID error handling
CREATE EXTENSION pgx_ulid;

-- Test invalid ULID string (too short)
ERROR:  invalid input syntax for type ulid: "01GV5PA9EQG7D82Q3Y4PKBZSY": invalid length

-- Test invalid ULID string (invalid character)
ERROR:  invalid input syntax for type ulid: "01GV5PA9EQG7D82Q3Y4PKBZSYU": invalid character

-- Test NULL handling
                test_name                | result 
---------------------------------------+--------
 NULL to ulid conversion returns NULL  | t
(1 row)

DROP EXTENSION pgx_ulid;
